microshift.sh

```diff
@@ -53,7 +53,25 @@ ssh-keygen -t ecdsa -b 521 -N "" -f id_ecdsa_crc -C "core"
 # createrepo package is required to create localrepo for microshift and it's dependenices
 # yum-utils package is required for reposync utility to synchronize packages of a remote DNF repository to a local directory
 function configure_host {
-    sudo dnf install -y git osbuild-composer composer-cli ostree rpm-ostree \
+    # https://github.com/openshift/microshift/blob/release-4.15/scripts/image-builder/configure.sh#L19-L35
+    source /etc/os-release
+    if [ "${ID}" == "rhel" ] && [ "${VERSION_ID}" == "9.3" ]; then
+        # osbuild from COPR to install version that:
+        # 1. can build rhel-93 images
+        # 2. doesn't have issues with unexpected mirror's RPM verification
+        #    (like the one in 9.3 beta at the moment of writing this comment)
```

Comment:
This comment mentions 9.3 beta, but 9.3 has been released 2 months ago. Is this still an issue?

Reply:
@cfergeau yes it is still same, I just took copy same content so can update it.

Reply:
I can check microshift team if it is there and link it to this one.

Reply:
@cfergeau added the link.